src/app/components/Loop/index.tsx

```diff
@@ -0,0 +1,425 @@
+            <Radio.Button value="b" onClick={this.setLoopInType}>
+              Loop In
+            </Radio.Button>
+          </Radio.Group>
+        </div>
+        <div className="Loop">
+          <InputLoopAddress
+            setLoop={this.props.setLoop}
+            setLoopIn={this.props.setLoopIn}
+            error={error}
+            initialUrl={this.props.url}
+            type={loopType}
+          />
+          {loopTerms.swap_fee_base !== '' && (
+            <Tooltip
+              overlayClassName="Loop-terms-tip"
+              title={`
+                Base fee: ${loopTerms.swap_fee_base} sats |
+                Fee rate: ${loopTerms.swap_fee_rate} sats |
+                Prepay amt: ${
+                  loopTerms.prepay_amt === undefined ? '1337' : loopTerms.prepay_amt
+                }
+                Min Swap amt: ${loopTerms.min_swap_amount} sats |
+                Max Swap amt: ${loopTerms.max_swap_amount} sats |
+                CLTV delta: ${loopTerms.cltv_delta}
+              `}
+              placement="topRight"
+              arrowPointAtCenter
+            >
+              <br />
+              <Button
+                className="Loop-form-advancedToggle"
+                onClick={this.toggleAdvanced}
+                type="primary"
+                ghost
+              >
+                {this.state.advanced === true
+                  ? 'Hide advanced fields'
+                  : 'Show advanced fields'}
+              </Button>
+            </Tooltip>
+          )}
+          <Form className="Loop" layout="vertical">
+            {url !== null && loopType === 'Loop Out' && (
+              <Form.Item>
+                <Dropdown overlay={loopOutMenu}>
+                  <a className="ant-dropdown-link" href="#">
+                    Select Channel
+                  </a>
+                </Dropdown>
+              </Form.Item>
+            )}
+            {url !== null && loopType === 'Loop In' && (
+              <Form.Item>
+                <Dropdown overlay={loopInMenu}>
+                  <a className="ant-dropdown-link" href="#">
+                    Select Channel
+                  </a>
+                </Dropdown>
+              </Form.Item>
+            )}
+            {advanced === true && loopType === 'Loop Out' && (
+              <Form.Item>
+                <Input
+                  type="text"
+                  size="small"
+                  onChange={this.handleChangeDestination}
+                  placeholder="off-chain address"
+                  autoFocus
+                />
+              </Form.Item>
+            )}
+            {advanced === true && loopType === 'Loop Out' && (
+              <Form.Item label="">
+                <Input
+                  size="small"
+                  onChange={this.handleChangeSwapRoutingFee}
+                  placeholder="swap routing fee"
+                  autoFocus
+                />
+              </Form.Item>
+            )}
```

Comment:
Same as above, these are mostly the same

Reply:
removed